src/browser/BrowserSettingsWidget.cpp

```diff
@@ -42,8 +42,9 @@ BrowserSettingsWidget::BrowserSettingsWidget(QWidget* parent)
 
     m_ui->extensionLabel->setOpenExternalLinks(true);
     m_ui->extensionLabel->setText(
-        tr("KeePassXC-Browser is needed for the browser integration to work. <br />Download it for %1 and %2 and %3. %4")
+        tr("KeePassXC-Browser is needed for the browser integration to work. <br />Download it for %1 and %2 and %3 and %4. %5")
```

Comment:
```suggestion
        tr("KeePassXC-Browser is needed for the browser integration to work. <br />Download it for %1, %2, %3, %4, %5")
```
Or maybe split this to one line per link?